Server/src/main/java/local/AssetDAOImpl.java

```diff
@@ -232,6 +232,31 @@ public void setAssetUpdate(int assetID) {
      */
     @Override
     public Asset createAssetFromQueryResult(ResultSet assetsQuery) throws SQLException {
+        Asset newAsset = new Asset();
+        newAsset.setId(assetsQuery.getInt("asset_id"));
+        newAsset.setName(assetsQuery.getString("name"));
+        newAsset.setAssetTypeID(assetsQuery.getString("asset_type_id"));
+        newAsset.setDescription(assetsQuery.getString("description"));
+        newAsset.setLocation(assetsQuery.getString("location"));
+        newAsset.setCategory(assetsQuery.getString("category"));
+        newAsset.setManufacturer(assetsQuery.getString("manufacturer"));
+        newAsset.setSite(assetsQuery.getString("site"));
+        newAsset.setSerialNo(assetsQuery.getString("sn"));
+        newAsset.setRecommendation(assetsQuery.getString("recommendation"));
+        newAsset.setAssetTypeName(assetsQuery.getString("asset_type.name"));
+        //newAsset.setAssetInfo(createAssetInfo(newAsset.getId()));
+        return newAsset;
+    }
```

Comment:
since this is used exclusively in the ui module, it should be in the external package of the ui module not in the server module

Reply:
createAssetFromQueryResult(ResultSet assetsQuery)

This method is used by 2 other methods in AssetDAOImpl.java in the Server Module.